compat/vcbuild/vcpkg_install.bat

```diff
@@ -34,6 +34,12 @@ REM ================================================================
 	@FOR /F "delims=" %%D IN ("%~dp0") DO @SET cwd=%%~fD
 	cd %cwd%
 
+	SET arch=x64-windows
```

Comment:
Is this really the correct thing to do? I'd rather have this as independent of the current architecture as possible.